### PR TITLE
ecore_x: Change EAPI symbols for ECORE_X_API

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

**Comment by Coquinho** [comment]
@joaoantoniocardoso it should be like this?

File: src/lib/ecore_x/ecore_x_api.h [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# ifndef ECORE_X_STATIC
+#  ifdef ECORE_X_BUILD
+#   define ECORE_X_API __declspec(dllexport)
+#  else
+#   define ECORE_X_API __declspec(dllimport)
+#  endif
+# else
+#  define ECORE_X_API
+# endif
+# define ECORE_X_API_WEAK
```
Comment (Coquinho) [inline src/lib/ecore_x/ecore_x_api.h:18]: @felipealmeida do we need a `weak` at `ecore_x`? This seems to be only used here at this file.

Comment (JPTIZ) [reply src/lib/ecore_x/ecore_x_api.h:18]: And there's no `WAPI` on the original (master's) code.

**Comment by Coquinho** [comment]
@JPTIZ this may be changed later

**Pull request merged**